python/services/github_service.py

```diff
@@ -376,6 +396,35 @@ def get_paginated_list_of_team_repositories(self, team_name: str, after_cursor:
             "after_cursor": after_cursor
         })
 
+    def fetch_all_repositories_in_org(self) -> list[dict[str, Any]]:
+        """A wrapper function to run a GraphQL query to get the list of repo names in the organisation
+
+        Returns:
+            list: A list of the organisation repos names
+        """
+        has_next_page = True
```

Comment:
Think we could default this to False? 🤔 When I was trying to implement high level integration tests, logic that defaulted to an infinite loop became quite a problem since you had to do low level stubbing to run the code

I believe defaulting it to break after the first iteration would help with implementing some integration tests

Reply:
May mean that the loop logic needs to be re-written though - which understandably, you may not want to do at the moment - so happy for it to be left 👍